tasks/publish-index-image/tests/test-publish-index-image.yaml

```diff
@@ -56,7 +58,8 @@ spec:
               #!/usr/bin/env sh
               set -eux
 
-              internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers)"
+              internalRequest="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
```

Comment:
The test should also test that the second IR was also called correctly. And also, we should add a second test where targetIndex already has a timestamped tag. (Once this is fixed in the task.)

Reply:
fixed.